substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1255,14 +1257,50 @@ protected int buildImage(List<String> javaArgs, LinkedHashSet<Path> bcp, LinkedH
 
         int exitStatus = 1;
         try {
+            Path argsFile = null;
```

Comment:
Move declaration of `argsFile` into the scope of `if (config.useJavaModules()) {`.

Reply:
ok